cmd/kubelet/app/server.go

```diff
@@ -261,8 +261,13 @@ func UnsecuredKubeletConfig(s *options.KubeletServer) (*KubeletConfig, error) {
 // The kcfg argument may be nil - if so, it is initialized from the settings on KubeletServer.
 // Otherwise, the caller is assumed to have set up the KubeletConfig object and all defaults
 // will be ignored.
-func Run(s *options.KubeletServer, kcfg *KubeletConfig) error {
-	var err error
+func Run(s *options.KubeletServer, kcfg *KubeletConfig) (err error) {
+	defer func() {
```

Comment:
This should mostly work, except when we hit some weird scoping issues. I'm ok with this for now, although a wrapper function would have been ideal.